packages/apps/spaces/package.json

```diff
@@ -45,6 +45,10 @@
     "react-toastify": "^9.1.1",
     "react-use-websocket": "^4.2.0",
     "react-virtual": "^2.4.10",
+    "react-time-ago": "^7.2.1",
+    "react-moment": "^1.1.3",
```

Comment:
moment is slow and heavy, let's use date-fns instead, it's already in the dependencies